assets/javascripts/discourse/connectors/after-create-topic-button/new-topic-dropdown.hbs

```diff
@@ -1,3 +1,7 @@
-{{#if currentUser}}
-  {{new-topic-dropdown category=category}}
+{{#if (and currentUser (not createTopicDisabled))}}
+  {{new-topic-dropdown
+    category=category
+    canCreateTopic=canCreateTopic
+    createTopicDisabled=createTopicDisabled
```

Comment:
this is always going to be true right?

Reply:
Not sure if I understood but given what I tested if the user is in a restricted group it will not render the dropdown.

In core createTopicDisabled is used with unless

Reply:
is `new-topic-dropdown` component using those arguments? 🤔